### PR TITLE
Add singlestore docs

### PR DESCRIPTION
**Pull request by tonykau1**
**Check List**
- [ ] Tests has been run in packages where changes made if available
- [ ] Linter has been run for changed code
- [ ] Tests for the changes have been added if not covered yet
- [ ] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: docs/content/Configuration/Databases/SingleStore.mdx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+ - Consolidated and flexible security model via tokens 
+
+
+## Prerequisites
+
+- The hostname for the [SingleStore][singlestore] workspace
+- The username/password for the [SingleStore][singlestore] workspace
+- The name of the database to use from the [SingleStore][singlestore] workspace
+
+## Setup <--{"id" : "Setup"}-->
```
Comment (hassankhan) [inline docs/content/Configuration/Databases/SingleStore.mdx:23]:
```suggestion
## Setup
```

File: docs/content/Configuration/Connecting-to-the-Database.mdx [hunk 2]
```diff
@@ -108,6 +108,11 @@ stores][ref-config-multi-data-src] out of the box.
     imageUrl="https://static.cube.dev/icons/sqlite.svg"
     title="SQLite"
   />
+  <GridItem
+    url="databases/singlestore"
+    imageUrl="https://ucarecdn.com/d68f15ae-0bd6-479f-b70c-742e14ce039e/"
```
Comment (hassankhan) [inline docs/content/Configuration/Connecting-to-the-Database.mdx:113]: Will get you a nicer logo for this

**Review by hassankhan — CHANGES_REQUESTED** [review]

File: docs/content/Configuration/Databases/SingleStore.mdx [hunk 3]
```diff
@@ -0,0 +1,149 @@
+---
+title: SingleStore
+permalink: /config/databases/singlestore
+---
+
```
Comment (hassankhan) [inline docs/content/Configuration/Databases/SingleStore.mdx:5]: Can we add a banner here saying the SingleStore driver isn't supported by us? 

Here's one from the Firebolt page as an example:

```

<WarningBox>

The driver for Firebolt is supported by its vendor. Please report any issues to
their [Help Center][firebolt-help].

</WarningBox>
```

Comment (tonykau1) [reply docs/content/Configuration/Databases/SingleStore.mdx:5]: Moved the MySQL notice to the top of the page for better visibility.

**Comment by igorlukanin** [comment]
Closing since we now already have SingleStore docs: https://cube.dev/docs/product/configuration/data-sources/singlestore

**Pull request closed without merging**